tensorflow/lite/micro/tools/make/pigweed_download.sh

```diff
@@ -32,7 +32,13 @@
 set -e
 
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
-ROOT_DIR=${SCRIPT_DIR}/../../../../..
+if [ ! -z ${2} ]
+then
```

Comment:
let's add a comment for why we need this in all the download files.

Reply:
Added a comment. Please let me know if you find the added comment is helpful. Thank you.